src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Common.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="12.0" DefaultTargets="PublishOutputLeg" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I wouldn't call this Common, but instead make it match the name of the package.

Reply:
Updated.